spec/forms/state_file/md_tax_refund_form_spec.rb

```diff
@@ -123,11 +156,59 @@
       end
 
       context "missing account holder name" do
-        let(:account_holder_name) { nil }
+        let(:account_holder_first_name) { nil }
+        let(:account_holder_last_name) { nil }
         it "is not valid" do
           form = described_class.new(intake, params)
           expect(form).not_to be_valid
-          expect(form.errors).to include :account_holder_name
+          expect(form.errors).to include :account_holder_first_name
+          expect(form.errors).to include :account_holder_last_name
+        end
+      end
+
+      context "missing joint account holder name" do
+        let(:joint_account_holder_first_name) { nil }
+        let(:joint_account_holder_last_name) { nil }
```

Comment:
it looks like you're testing the following cases for these params:
- missing but not required 
- required and valid
- required and invalid 

but not testing the case where they're required and missing